app/assets/stylesheets/profile.scss

```diff
@@ -0,0 +1,10 @@
+#profile-posts {
+	font-size: 20px;
+	border: 1px solid black;
+	width: auto;
+	height: auto;
+}
+
+.profile-description {
```

Comment:
check [sublime-to-be-done](https://github.com/tiffon/sublime-to-done) in the future, a lot more convenient! :heart_eyes: